### ISSUE TITLE
[Output] Update the output class ``resultData.py`` to provide a nice string representation.

### ISSUE THREAD

**Issue by MaLiN2223** [tea-lang-org/tea-lang#63]

**Comment by MaLiN2223** [comment]
@emjun, do you mind providing some more context? What do we mean by 'nice' representation? Is there any specification or do you have any idea how it should look like?
In the meantime, I will refactor the code so we can change it easier.

**Comment by MaLiN2223** [comment]
@emjun, I think I will need your guidance on the new api, I am unable to figure out how to reach ResultData object with the new api. 
So far, this worked:

```
from tea.api import Tea
variables = [
    {"name": "So", "data type": "nominal", "categories": ["0", "1"]},
    {"name": "Prob", "data type": "ratio", "range": [0, 1]}, 
    {"name": "Ineq", "data type": "ratio"},
]
design = {
    "study type": "observational study",
    "contributor variables": ["So", "Prob"],
    "outcome variables": ["Prob", "Ineq"],
}
tea = Tea(variables, design)
tea.load_data("./examples/US_crime/UScrime.csv")
hypo = tea.hypothesize("Ineq ~ Prob")
```

However, I don't know how to proceed further to see the ResultData, seems like the new hypothesize does't use it.

**Comment by emjun** [comment]
Yeah, I haven't finished implementing the new hypothesize, so maybe that's why. What are you wanting to access ResultData for? Maybe I can provide support for the specific use case then make it more general.

**Comment by MaLiN2223** [comment]
I am working on the task to make ResultData's print nicer (#64 ) and want to test my changes end to end

**Comment by emeryberger** [comment]
FWIW I am using the “rich” library for Scalene and it’s pretty awesome. You can easily generate tables, pretty printing, console, markdown and HTML output...

https://pypi.org/project/rich/
